src/main/kotlin/blackjack/domain/BlackJackBuilder.kt

```diff
@@ -0,0 +1,33 @@
+package blackjack.domain
+
+fun introduce(block: BlackJackBuilder.() -> Unit): BlackJack {
+    return BlackJackBuilder().apply(block).build()
+}
+class BlackJackBuilder {
+    private lateinit var cardDeck: CardDeck
+    private lateinit var participants: Participants
+    fun cardDeck(cards: List<Card>) {
+        cardDeck = CardDeck(cards)
+        cardDeck.shuffle()
+    }
+
+    fun participants(block: ParticipantsBuilder.() -> Unit) {
+        participants = ParticipantsBuilder().apply(block).build()
+    }
+
+    fun draw() {
+        participants.all().map {
```

Comment:
```suggestion
        participants.all().forEach {
```
무언가를 변환하는 것이 아니라면 forEach를 활용해보면 어떨까요?